cmd/utils/flags.go

```diff
@@ -858,6 +858,18 @@ var (
 		Name:  "rpc.getlogs.maxrange",
 		Usage: "Limit max fetched block range for `eth_getLogs` method",
 	}
+
+	// Clique shadowfork
+	ShadowforkFlag = cli.StringFlag{
+		Name:  "clique.shadowfork",
+		Usage: "Schedule a shadow fork for a given signer to takeover at a given height",
```

Comment:
The new signer identity will be configured somewhere? Or will it be a hardcoded test account?

It looks like we'd need to provide a block height and an address here? But doesn't the node need a private key (keystore) to sign blocks?

Reply:
usage is something like `--clique.shadowfork "<height>,<signerAddr>"`

> But doesn't the node need a private key (keystore) to sign blocks?

Yeap, signer node will need to be configured thru the existing flags like `--unlock`